src/server_prep.py

```diff
@@ -0,0 +1,103 @@
+import argparse
+import os.path
+import uuid
+from urllib.parse import urlparse
+
+import pyspark.sql.functions as F
+from pyspark.sql import SparkSession, Window
+from pyspark.sql.functions import udf
+from pyspark.sql.types import StringType
+
+from schemes import multimedia_scheme
+from utils.utils import truncate_folder
+
+BATCH_SIZE = 10_000
+
+
+@udf(returnType=StringType())
+def get_server_name(url: str):
+    return urlparse(url).netloc
+
+
+@udf(returnType=StringType())
+def get_uuid():
+    return str(uuid.uuid4())
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Convert TSV to Parquet')
+
+    parser.add_argument('path', metavar='path', type=str, help='the path to the tsv file')
+    parser.add_argument('output_path', metavar='output_path', type=str, help='the path to the output folder')
```

Comment:
```suggestion
    parser.add_argument('output_path', metavar='output_path', type=str, help='the path to the output folder (folder for download components (e.g., server batches and image folder))')
```
Folder containing image folder and all the other download components (called in the MPI `py` files in `src` is the output folder, right?